src/components/calculator.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+export default function Calculator() {
+  return (
+    <div className="calculator_container">
+      <p>0</p>
+      <div className="number_wrapper">
+        <div className="row_container">
+          <button type="button">AC</button>
+          <button type="button">+/-</button>
+          <button type="button">%</button>
+          <button type="button">/</button>
+        </div>
+        <div className="row_container">
+          <button type="button">9</button>
+          <button type="button">8</button>
+          <button type="button">7</button>
+          <button type="button">x</button>
+        </div>
+        <div className="row_container">
+          <button type="button">6</button>
+          <button type="button">5</button>
+          <button type="button">4</button>
+          <button type="button">-</button>
```

Comment:
- [ ] PLease nsure that your design matches the given design as this a requirement

Given design
![image](https://user-images.githubusercontent.com/81932478/218873342-e74e07b3-0f10-4d81-b434-8992234c84af.png)

Your design
![image](https://user-images.githubusercontent.com/81932478/218873403-0ff60322-331a-4045-957d-a994e98f8a26.png)

- [ ] KIndly follow the given colors and layout of the components